index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Startup Matchmaker</title>
+  <style>
```

Comment:
Hey Robert, I tried to put the background image into my stylesheet and I can't get the image to show up now... any suggestions?